app/Helpers/render/achievement.php

```diff
@@ -91,12 +94,15 @@ function renderAchievementCard(int|string|array $achievement, ?string $context =
     $sanitizedGameTitle = str_replace("\n", '', $renderedGameTitle);
     $sanitizedGameTitle = trim(html_entity_decode($sanitizedGameTitle, ENT_QUOTES, 'UTF-8'));
 
+    $titleClassName = '';
     $renderedType = null;
     if ($type) {
         $renderedType = str_replace("\n", '', Blade::render(
             '<x-achievement.thin-type-indicator :type="$type" />',
             ['type' => $type]
         ));
+
+        $titleClassName = 'max-w-[190px]';
```

Comment:
Is 190 perhaps too conservative?

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/d8f10eb4-b522-4e75-925b-b684c01aa6df)

Reply:
Updated in latest to take the indicator label type into account when setting the width.